pkg/engine/health/health.go

```diff
@@ -121,6 +121,12 @@ func IsHealthy(obj runtime.Object) error {
 		}
 		return fmt.Errorf("pod %q is not running yet: %s", obj.Name, obj.Status.Phase)
 
+	case *corev1.Namespace:
```

Comment:
Why do we care about namespace status here? Asking because namespaces aren't managed by KUDO.

Reply:
Well, there's nothing that stops you from deploying a namespace with KUDO :( 

And it makes sense to have it here. We use it in the "VerifyInstallation" of the namespace.go

Reply:
Got it! Thanks for the explanation.